src/window/editor/scenario_events.c

```diff
@@ -15,8 +15,11 @@
 #include "scenario/editor_events.h"
 #include "scenario/property.h"
 #include "scenario/scenario_events_controller.h"
+#include "widget/input_box.h"
```

Comment:
Is this used?

Reply:
Nope. Good spot. Forgot to remove the include after looking to use the input box at one point.
Removed.